meta/contao/skeleton-bundle/en.yml

```diff
@@ -0,0 +1,5 @@
+en:
+    title: Skeleton Bundle for Contao
+    description: >
+        This bundle adds rapid development tools and code generators to Contao 4.
```

Comment:
It actually does neither of that. 🤷‍♂

Reply:
yeah, I thought this is the maker bundle 😂